src/app/(BottomTabNavigation)/AllCases/EligibilityForm/[caseUid].tsx

```diff
@@ -19,39 +35,67 @@ export default function EligibilityForm() {
       });
     }
   };
+
   return (
-    <View style={styles.container}>
+    <ScrollView style={styles.container}>
+      {caseData && (
+        <>
+          <Image style={styles.image} source={{ uri: caseData.imageUrl }} />
+          <Text style={styles.title}>{caseData.title}</Text>
+        </>
+      )}
+
       <View style={styles.textContainer}>
         <View style={styles.insideContainer}>
-          <Text>The eligibility requirements are as follows:</Text>
-          <View>
-            <Text>Requirement 1</Text>
-            <Text>Requirement 2</Text>
-            <Text>Requirement 3</Text>
-          </View>
+          <Text style={styles.info}>
+            You must meet every requirement to be eligible for this
+            class-action.
+          </Text>
+          <LineBig />
+          <Text style={styles.reqs}>
+            <Checkbox />
+            This is the first requirement that you should meet to file a claim
+            <LineSmall />
+          </Text>
+          <Text style={styles.reqs}>
+            <Checkbox />
+            This is the next requirement that you should meet to file a claim
+            <LineSmall />
+          </Text>
+          <Text style={styles.reqs}>
+            <Checkbox />
+            This is the last requirement that you should meet to file a claim
+            <LineSmall />
+          </Text>
+          <Text style={styles.reqs}>
+            <Checkbox />
+            This is the last requirement that you should meet to file a claim
+            <LineSmall />
+          </Text>
+          <Text style={styles.reqs}>
+            <Checkbox />
+            This is the last requirement that you should meet to file a claim
+            <LineSmall />
+          </Text>
           <Text>Do you meet the following requirements?</Text>
         </View>
       </View>
       <View style={styles.buttonsContainer}>
-        <View style={styles.buttonWrapperTop}>
-          <TouchableOpacity
-            style={[styles.button, styles.buttonTop]}
-            onPress={() => updateEligibility(Eligibility.ELIGIBLE)}
-          >
-            <Text>Yes, I am Eligible</Text>
-            <RightArrow />
-          </TouchableOpacity>
-        </View>
-        <View style={styles.buttonWrapperBottom}>
-          <TouchableOpacity
-            style={[styles.button, styles.buttonBottom]}
-            onPress={() => updateEligibility(Eligibility.INELIGIBLE)}
-          >
-            <Text style={styles.buttonBottomText}>No, I'm not Eligible</Text>
-            <Text style={styles.buttonBottomText}>X</Text>
-          </TouchableOpacity>
-        </View>
+        <TouchableOpacity
+          style={[styles.button, styles.buttonTop]}
```

Comment:
Keep in mind that the "Yes I Do" option should be greyed out until all boxes are checked. It might be easiest to create a state variable on this screen that determines this. The setState function will be passed to each list item component.